### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by blerude** [LoopyFruits/prj-rev-bwfs-tea-cozy#4]
Grade: Exceeds Expectations

Awesome job on this project! You've done a great job incorporating the necessary elements on the page and of making your code easily digestible. Your comments and the consistency in how you space/align/indent your code both make this project easy to understand. Those habits will only become more valuable as your projects grow in size and complexity, and as you begin collaborating with others. 

You also use classes and ids effectively, although I'd pay attention to the names that you give. You want those names to be indicative of what the element does, so names like 'display' or id names that are also used as class names don't give any insight about what that element's specific purpose is. Think about that moving forward! Again, great job on this project and congratulations!

**Comment by LoopyFruits** [comment]
Yeah I often don't know what to name things, I was thinking like a display shelf in a store, based it around that theme. But coming up with unique names for everything can be challenging.
